change/@ni-nimble-blazor-ad0b7e2c-d4ff-4837-b0c0-ec203eee88f7.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Added logic to stop NimbleTabs from seeing `change` event from children components. Followed the pattern FluentUI Blazor uses for their Tab component.",
```

This comment ends up in the changelog so we usually try to make it focused more on the impact to clients than on implementation details. How about something like this instead?
```suggestion
  "comment": "Fix bug where NimbleTab would respond to `change` event from children components, causing tabs to change.",
```

It would be good to update your PR title to be more descriptive too e.g. "Fix bug with BlazorTabPanel event handling"

---

Fixed in the latest push.